bayesian/api/api_v2.py

```diff
@@ -198,8 +198,10 @@ def post():
         input_json: Dict = request.get_json()
         ecosystem: str = input_json.get('ecosystem')
         user_agent = request.headers.get('User-Agent', None)
-        manifest_hash = str(request.headers.get('manifest_hash', None))
-        request_id = request.headers.get('request_id', None)
+        manifest_hash = request.args.get('utm_content', None)
+        source = request.args.get('utm_source', None)
+        request_id = request.headers.get('X-Request_Id', None)
+        telemetry_id = request.headers.get('X-Telemetry_Id', None)
```

Comment:
Instead of extracting it one by one and passing it as separate argument, why not just pass the `request.headers` to `create_component_bookkeeping` and extract the needed fields?

Reply:
Done.